src/hostname/hostnamectl.c

```diff
@@ -108,6 +109,17 @@ static const char *os_support_end_color(usec_t n, usec_t eol) {
         return NULL;
 }
 
+static char *wrap_ansi_seq(const char *ansi_seq) {
+        _cleanup_free_ char *ansi_color = NULL;
+        int r;
+
+        r = asprintf(&ansi_color, "\x1B[%sm", ansi_seq);
+        if (r > 0)
```

Comment:
if you want to concat strings, please use strjoin() rather than asprintf(). Because asprintf() with its format strings is actually irritatingly slow. not that the performance mattered much here, but we try to stick to the rule that string concat is a job for strjoin()